lib/buttons_bar.dart

```diff
@@ -0,0 +1,66 @@
+import 'plain_text.dart';
+import 'ui_constants.dart';
+import 'package:flutter/material.dart';
+
+
+class ButtonsBar extends StatefulWidget {
+  const ButtonsBar({super.key});
+
+  @override
+  State<ButtonsBar> createState() => _ButtonsBar();
+}
+
+class _ButtonsBar extends State<ButtonsBar> {
+
+  int _counter = 0;
+  static const String playTrailer = 'Play trailer';
+
+  void _incrementCounter() {
+    setState(() {
+      _counter++;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceAround,
+      children: [
+        Container(
+          decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(
+                UIConstants.buttonBorderRadius),
+            border:
+            Border.all(color: UIConstants.contrastColor),
+          ),
+          child: const Padding(
+            padding: UIConstants.paddingPlayButton,
```

Comment:
Super minor: You could use the `padding` property of the `Container` instead of using a separate `Padding` widget for this!

Reply:
done!